### PR TITLE
De cleanups/20241010/v4

### PR DESCRIPTION
**Pull request by inashivb**
Previous PR: https://github.com/OISF/suricata/pull/11957

Changes since v3:
- handled typos
- made the legacy setting clear as a part of config message

### CODE REVIEW

**Review by victorjulien — APPROVED** [review]

File: suricata.yaml.in [hunk 1]
```diff
@@ -1702,12 +1702,12 @@ detect:
...
   # the grouping values above control how many groups are created per
-  # direction. Port whitelisting forces that port to get its own group.
+  # direction. Port priority setting forces that port to get its own group.
   # Very common ports will benefit, as well as ports with many expensive
   # rules.
   grouping:
-    #tcp-whitelist: 53, 80, 139, 443, 445, 1433, 3306, 3389, 6666, 6667, 8080
-    #udp-whitelist: 53, 135, 5060
+    #tcp-priority-ports: 53, 80, 139, 443, 445, 1433, 3306, 3389, 6666, 6667, 8080
+    #udp-priority-ports: 53, 135, 5060
```
Comment (catenacyber) [inline suricata.yaml.in:1710]: Change to the config without documentation ?

Comment (victorjulien) [reply suricata.yaml.in:1710]: Looks like it was never documented in the first place. Not a blocker here, but would like to see a doc ticket @inashivb

**Comment by victorjulien** [comment]
Merged in #11965, thanks!

File: suricata.yaml.in (hunk 1 above)
Comment (inashivb) [reply suricata.yaml.in:1710]: Thank you, Philippe! 🙇🏽‍♀️

**Pull request closed without merging**